trace-dispatcher/src/Cardano/Logging/Trace.hs

```diff
@@ -248,6 +268,7 @@ withDetails (Trace tr) = Trace $
         (lc, Left e) -> (lc, Left e))
       tr
   where
+    {-# INLINE process #-}
```

Comment:
Only top-level definitions can be marked `INLINE`. As it's ignored here, can we remove it?

Reply:
Done